Gems/Atom/Tools/MaterialEditor/Code/CMakeLists.txt

```diff
@@ -133,6 +133,9 @@ ly_add_target_dependencies(
     DEPENDENCIES_FILES
         tool_dependencies.cmake
         Source/Platform/${PAL_PLATFORM_NAME}/tool_dependencies_${PAL_PLATFORM_NAME_LOWERCASE}.cmake
+    # The Material Editor needs the LyShine "Tools" gem variant for the custom LyShine pass
+    DEPENDENT_TARGETS
+        Gem::LyShine.Tools
```

Comment:
The material editor itself does not need LyShine by default. LyShine was added to the automated testing project. My understanding was that the material editor would be designated as a tool and load the projects tool variants.

Reply:
I am leaving this up to the Atom team to update the MaterialEditor to use the `Tools` variant instead.